stage #1/module #1. Exceptions/lesson #1. Overview/owerview.md

```diff
@@ -0,0 +1,45 @@
+# Exceptions Overview
+
+## Agenda
```

Comment:
Could you use the link for your points from agenda?
[Materials](#materials)
## Materials

Reply:
yes, thanks